### PR TITLE
Knitting with CSS Grid post

### PR DESCRIPTION
**Pull request by csteinert**
*WIP*

Markup and Conclusion sections are unfinished. 

Feedback welcome.

### CODE REVIEW

File: _posts/2017-10-13-quilting-with-css-grid.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Now all that's left is to position the grid items accordingly! The tricky part
+with this pattern is that the strips get progressively wider/taller as they are
+added. Only the first two items exist within one column and row. All the other
+items span at least two rows or columns.
+
+
+
+
+
+I did that using the `span` notation in combination with the shorthand
```
Comment (kalongthewires) [inline _posts/2017-10-13-quilting-with-css-grid.md]: "I did that" - it is a bit unclear what "that" is.

File: _posts/2017-10-13-quilting-with-css-grid.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+}
+```
+
+
+
+
+Conclusion
+- Is this a realistic layout? No.
+- Was it fun? Yes.
+- What did I learn? Stuff.
```
Comment (kalongthewires) [inline _posts/2017-10-13-quilting-with-css-grid.md]: Liking the conclusion 😂

File: _posts/2017-10-13-quilting-with-css-grid.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+ has some great photos that show how this construction method works. The yarn I
+used to make the squares changes colors gradually, which makes it possible to
+see the order in which the rectangles were added.
+
+![Knit log cabin blanket square with numbers showing the order in which the sections were knit](https://farm5.staticflickr.com/4575/38414998926_d434047620_z.jpg)
+
+
+## The Markup
+
+I used [Codepen](http://codepen.io), my favorite online developer community, to
```
Comment (sludemann) [inline _posts/2017-10-13-quilting-with-css-grid.md]: ~~to~~

File: _posts/2017-10-13-quilting-with-css-grid.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+with this pattern is that the strips get progressively wider/taller as they are
+added. Only the first two items exist within one column and row. All the other
+items span at least two rows or columns.
+
+
+
+
+
+I did that using the `span` notation in combination with the shorthand
+declarations (`grid-column` and `grid-row`) that allow us to tell an item where
```
Comment (sludemann) [inline _posts/2017-10-13-quilting-with-css-grid.md]: I might go with just commas instead of parentheses here, but both work.

**Comment by chriswhawkins** [comment]
I don't have anything to add beyond Kim and Sean's notes, but this is great! It's fun, readable, and informative 👍 Can't wait to see the ending :)

File: _posts/2017-10-13-quilting-with-css-grid.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+Once each piece of the pattern has the correct location assigned to it, the
+blanket is complete!
+
+
+### Conclusion
+
+While this is not an example of a layout that I'd be able to use anywhere else,
+it was a fun and challenging way to sharpen my grid-writing skills. I learned
+more about `span` notation in grid and discovered shorthand properties that I
+wasn't didn't already know about.
```
Comment (chriswhawkins) [inline _posts/2017-10-13-quilting-with-css-grid.md]: ➖  wasn't

File: index.html [hunk 6]
```diff
@@ -253,7 +253,7 @@ <h4>Reed Dunkle
 
       <div class="team-member-info">
         <h4>Mandy Kendall
-          <small>Developer Apprentice</small>
+          <small>Developer</small>
```
Comment (chriswhawkins) [inline index.html]: 👍

**Pull request merged**